src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -404,6 +420,12 @@ protected int doWork() {
             log.info("no successfully lifted variants");
         }
 
+        if (RECOVER_SWAPPED_REF_ALT) {
+            log.info(totalRecoveredBySwapRefAlt, " variants were lifted by swapping REF/ALT alleles.");
+        } else {
+            log.info(totalNotRecoveredSwapRefAlt, " variants with a swapped REF/ALT were identified, but were not recovered.  See RECOVER_SWAPPED_REF_ALT.");
```

Comment:
This should be a warning, I think.

Reply:
I dont feel all that strongly about this, but I am not sure I agree.  I assume you only really think the warning matters for line 426?  I flipped that to warn() and slightly updated the verbiage.